hazelcast/src/test/java/com/hazelcast/test/mocknetwork/MockServer.java

```diff
@@ -65,24 +65,24 @@
     private final Node node;
 
     private final ScheduledExecutorService scheduler;
-    private final IOService ioService;
+    private final ServerContext serverContext;
     private final ILogger logger;
 
     private volatile boolean live;
 
     private final ServerConnectionManager mockConnectionMgr;
     private final AggregateServerConnectionManager mockAggrEndpointManager;
 
-    MockServer(IOService ioService, Node node, TestNodeRegistry testNodeRegistry) {
-        this.ioService = ioService;
+    MockServer(ServerContext serverContext, Node node, TestNodeRegistry testNodeRegistry) {
```

Comment:
minor: there is a leftover reference to `NodeIOService` in `MockServer:134`

Reply:
Fixed